Zend/zend_execute.c

```diff
@@ -1018,13 +1025,7 @@ static zend_never_inline zval* zend_assign_to_typed_prop(zend_property_info *inf
 	}
 
 	if (UNEXPECTED(info->flags & ZEND_ACC_READONLY)) {
```

Comment:
I'm not sure whether it's still desirable to unset the flag unconditionally, or rather only after checking for the readonly flag

Reply:
I'd say so.